README.md

```diff
@@ -1,149 +1,18 @@
-```
-.\venv\Scripts\activate
-```
-#### <b>Step 3:Now we will see what are the list of packages that have already been installed using pip: </b>
-```
-pip list
-```
-#### <b>Step 4:Incase the pip version has been depercated, it is advisable to follow these steps to upgrade the pip:</b>
-```
-pip install --upgrade pip
-```
-#### <b>Step 5:To install Tensorflow </b>
-```
-pip install tensorflow
-```
-#### <b>Step 6:In case you want to exit the virtual environment just type in the command:</b>
-```
-deactivate
-```
-
-
-These commands will install Tensorflow on a Windows System with ease using pip. In case you are using an **Anaconda Navigator**, there are seperate set of commands which are explained here: 
-
-#### <b>Step 1: Open the Command Prompt and type in the command:</b>
-```
-conda info
-```
-This command will display all of the necessary details of Anaconda including the Active Environments and more
-#### <b>Step 2: Create an Environment on Conda using the following command:</b>
-```
-conda info
-```
-#### <b>Step 3:Type the Command:</b>
-```
-conda create -n [environment-name]
-```
-Let's say I want to name my environment as "TensorTest" so my command would be: 
-```conda create -n tensortest```
-#### <b>Step 4:Now run the following environment that you have just created:</b>
-```
-conda install tensorflow
-```
-
-Now we have seen how we can install Tensorflow on Windows using pip and conda. Now let's verify that we have installed Tensorflow successfully by running a simple Tensorflow Code: 
-#### <b>Step 1: Open the Command Prompt.</b>
-#### <b>Step 2: Type the following command to kickstart a Python Shell:</b>
-```
-python
-```
-#### <b>Step 3: Now type the following code:</b>
-```
-python
->>> import tensorflow as tf
->>> output=tf.constant("Hello World")
->>> sess=tf.Session()
->>> print(sess.run(output))
-'Hello World'
->>> sess.close()
-```
-We will see an output on the Command Prompt Screen which will verify that Tensorflow is now up and running on our system.
-
-
-
-
-
-## <strong>Tensorflow Installation on Linux</strong>
-
-### <b>Prerequisites</b>
-1. Python 3.3 or higher
-2. pip 19.0 or a higher version 
-3. At least 1GB of RAM
-
-### Installation
-#### <b>Step 1: Install python3-venv</b>
-```
-apt-get install python3-venv -y
-```
-
-#### <b>Step 2: Create and activate a Python virtual environment</b>
-```
-python3 -m venv venv
-source ./venv/bin/activate
-```
-output:
-```
-(venv) root@ubuntu:~#
-```
-
-#### <b>Step 3: Update PIP</b>
-```
-pip install -U pip
-```
-
-#### <b>Step 4: Update setuptools</b>
-```
-pip install -U setuptools
-```
-
-#### <b>Step 5: Install TensorFlow</b>
-```
-pip install tensorflow
-```
-
-*For more details, visit the official tensorflow documentation page : https://www.tensorflow.org/*
+** Refer [this](https://github.com/CropAi/Data-Modeling/blob/master/docs/transfer learning.md) for knowing how to use transfer learning for Crop Disease Detection. 
```

Comment:
This link doesnt show up properly

Reply:
Link to Transfer learning.md

Reply:
@gshanbhag525 Sir the link will work only after you will merge this pull request as currently there doesn't exist any docs and module folder in your repository.

Reply:
If you wish I could add the link of docs and modules folder of my repository for testing which you would have to change after merging the pull request.

Reply:
@Aman-Codes No. My point was not of opening and viewing the link.
It was there is a whitespace in between transfer and learning.md 
So it diturbs the the link patter and the link only gets tiil transfer. learning 
<img width="1680" alt="Screen Shot 2020-03-27 at 7 18 29 PM" src="https://user-images.githubusercontent.com/17478096/77762616-ceaefd00-705f-11ea-869b-3e3a7563f59f.png">

Reply:
@gshanbhag525 Thank you sir for pointing out that. Sir please guide me what should I do to rectify it shall I replace blankspace with %20 (hopefully that may work) or should I rename the transfer learning file to transfer_learning.md as in general it is not a good practice to add spaces while naming files.

Reply:
renamin it to transfer_learning.md

Reply:
@gshanbhag525 did the asked changes changes. Please review it and do let me know if any changes are required.